lib/developer_portal/app/controllers/developer_portal/admin/account/payment_details_base_controller.rb

```diff
@@ -74,4 +73,17 @@ def account_params
     allowed_fields = %i[name address1 address2 city country state phone zip first_name last_name]
     params.require(:account).permit(billing_address: allowed_fields)
   end
+
+  def update_billing_address
+    current_account.updating_payment_detail = true
```

Comment:
What is `updating_payment_detail` for?

Reply:
It was already there, so I decided not to touch it.
These tests clarify this... somewhat: https://github.com/3scale/porta/blob/e25ee3910441db42ca0255d9d97a62f9ea33b2d5/test/unit/account/credit_card_test.rb#L88-L98

On the other hand, it seems that this `payment_detail_conditions` thing only appears in this test :grimacing:  https://github.com/search?q=repo%3A3scale%2Fporta%20payment_detail_conditions&type=code 
Not sure it's still relevant.

Reply:
It was added here: https://github.com/3scale/system/commit/50b94d8467160a5ff806f2820ce70967d4048bfd 

But I've never seen this checkbox... So, I think we can remove. Probably, in another PR.

Reply:
I've changed my mind and I've removed it here: https://github.com/3scale/porta/pull/3696/commits/fa0dc1e39a23cc9af797f5779a92eb2b64a273b7

I found enough evidence (I think) to ensure this will not break anything. And I prefer to do it in this PR, so in case we need to roll it back or something, we don't have issues.

WDYT @jlledom ?

Reply:
Fine for me. I'm always for cleaning up.